backend/app/usecase/changelog/changelog.go

```diff
@@ -47,21 +48,31 @@ func (p Persist) GetChangeLog() ([]entity.Change, error) {
 }
 
 // GetLastViewedAt retrieves the last time the user viewed the change log
-// TODO(issue#613): fetch the last time the user viewed the change log from persistent storage.
-func (p Persist) GetLastViewedAt() *time.Time {
-	now := p.timer.Now()
-	return &now
+func (p Persist) GetLastViewedAt(user entity.User) (*time.Time, error) {
+	lastViewedAt, err := p.userChangeLogRepo.GetLastViewedAt(user)
+	if err == nil {
+		return &lastViewedAt, nil
+	}
+
+	switch err.(type) {
+	case repository.ErrEntryNotFound:
+		return nil, nil
```

Comment:
I think we should create relation here. `ViewChangeLog` function is responsible for updating the field.

Reply:
Please also add a test case for this.